Sphinx admin page

turizspace opened the pull request:
Build a simple admin page showing a table of users, and when one is clicked, showing their payments

https://github.com/user-attachments/assets/dcc06c27-e056-4317-9380-5b5a2389fa96

turizspace committed:
Sphinx admin page

Build a simple admin page showing a table of users, and when one is clicked, showing their payments

Evanfeenstra commented:
can you make the root and pubkey shorter like “039a…57d8”

- show a "child keys" column that shows the scids.length. 
- show "payments count" column
- show "active" column which is a date (use day.js to render a date like MM/DD/YYYY)

turizspace committed:
[Update] Sphinx admin page

Made the root and pubkey shorter
show a "child keys" column that shows the scids.length.
show "payments count" column
show "active" column which is a date

turizspace commented:
Made the changes requested. @Evanfeenstra 
https://github.com/user-attachments/assets/cfcb3fc2-db32-4fa9-b449-336358351665
